spec.html

```diff
@@ -36368,7 +36368,7 @@ <h1>The RegExp Constructor</h1>
       <ul>
         <li>is <dfn>%RegExp%</dfn>.</li>
         <li>is the initial value of the *"RegExp"* property of the global object.</li>
-        <li>creates and initializes a new RegExp object when called as a function rather than as a constructor. Thus the function call `RegExp(…)` is equivalent to the object creation expression `new RegExp(…)` with the same arguments.</li>
+        <li>creates and initializes a new RegExp object when called as a function rather than as a constructor (unless the only argument is already a RegExp object).</li>
```

Comment:
```suggestion
        <li>when called as a function, returns either a new RegExp object, or the argument itself if the argument is a RegExp.</li>
```
?

Reply:
I like this suggestion.

Reply:
Updated (but I kept the “only argument” wording to acknowledge the distinction between `RegExp(r) === r` and `RegExp(r, "") !== r`).